chainer_setup_build.py

```diff
@@ -268,3 +328,164 @@ def get_ext_modules():
 
     check_extensions(extensions)
     return extensions
+
+
+def _nvcc_gencode_options():
```

Comment:
Can you move this method to `install` directory and make a test? I have no idea to this method, but maybe it is enough to test outputs of all sported version of `nvcc` commands can be parsed.